content/attacks/posts/Create - 2023-11-23-HecoCrossChainBridge&HTX.md

```diff
@@ -0,0 +1,37 @@
+---
+date: 2023-11-23
+target-entities: Heco Cross Chain Bridge and HTX
+entity-types:  
+   - Exchange
```

Comment:
Add custodian and DeFi as an entity type, since HTX exchange is centralized, and Heco Bridge is decentralized

Reply:
Add or replace?